doc/DeveloperGuide.md

```diff
@@ -94,6 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should store record in human readable files.
+6. Should have intuitive command name and format
```

Comment:
Again this seems general. Perhaps `It should take no less than 1 hour to know the basic command`.